python/ray/serve/_private/http_proxy.py

```diff
@@ -190,11 +189,15 @@ def __init__(
             )
 
         def get_handle(deployment_name, app_name):
-            return serve.context.get_global_client().get_handle(
-                deployment_name,
-                app_name,
-                sync=False,
-                missing_ok=True,
+            return (
+                serve.context.get_global_client()
+                .get_handle(
+                    deployment_name,
+                    app_name,
+                    sync=False,
+                    missing_ok=True,
+                )
+                .options(use_new_handle_api=True)
```

Comment:
Just also want to note this is also done in the [proxy router](https://github.com/ray-project/ray/blob/master/python/ray/serve/_private/proxy_router.py#L68), but no harm setting it again

Reply:
got it